src/frontend/js/components/CourseGlimpse/CourseGlimpse.tsx

```diff
@@ -36,49 +36,49 @@ const messages = defineMessages({
 });
 
 export const CourseGlimpse = injectIntl(
-  (props: CourseGlimpseProps & InjectedIntlProps) => {
-    const { course, intl, organizationMain } = props;
-
-    return (
-      <a
-        className="course-glimpse course-glimpse--link"
-        href={course.absolute_url}
-        title={intl.formatMessage(messages.linkText, {
-          courseTitle: course.title,
-        })}
-      >
-        <div className="course-glimpse__media">
-          <img
-            src={course.cover_image}
-            alt={intl.formatMessage(messages.altText, {
-              courseTitle: course.title,
-            })}
-          />
-        </div>
-        <div className="course-glimpse__content">
-          <div className="course-glimpse__content__wrapper">
-            <p className="course-glimpse__content__title">{course.title}</p>
-            <p>{(organizationMain && organizationMain.title) || ''}</p>
-          </div>
+  ({
+    course,
+    intl,
+    organizationMain,
+  }: CourseGlimpseProps & InjectedIntlProps) => (
+    <a
+      className="course-glimpse course-glimpse--link"
+      href={course.absolute_url}
+      title={intl.formatMessage(messages.linkText, {
+        courseTitle: course.title,
+      })}
+    >
+      <div className="course-glimpse__media">
+        <img
+          src={course.cover_image}
+          alt={intl.formatMessage(messages.altText, {
+            courseTitle: course.title,
+          })}
+        />
+      </div>
+      <div className="course-glimpse__content">
+        <div className="course-glimpse__content__wrapper">
+          <p className="course-glimpse__content__title">{course.title}</p>
+          <p>{(organizationMain && organizationMain.title) || ''}</p>
```

Comment:
This prop looks like unused, it is set to `null` in `CourseGlimpseList` component. Did you keep to use it later ?

Reply:
Yes. I think this probably should come as a field on the `course` now (as we discussed with @sampaccoud previously).

I did not want to go and do that while we're in the middle of a large refactor on the backend side to avoid conflicts.